common/services/reference-data.js

```diff
@@ -49,14 +49,23 @@ function getLocationById(id) {
   return apiClient.find('location', id).then(response => response.data)
 }
 
-function getLocationsById(locations = []) {
-  const locationPromises = locations.map(locationId => {
-    return getLocationById(locationId).catch(() => false)
-  })
+function getLocationByAgencyId(agencyId) {
```

Comment:
Could also name the argument `nomisAgencyId` so you can use shorthand in the `getLocations` call.

Reply:
I was thinking of renaming the function `getLocationByNomisAgencyId` (and everywhere else it is called `agencyId` to be clearer, do you think this is worthwhile?

Reply:
I was also thinking the same thing. I think it's clearer to what it's referring to as these are tied directly to NOMIS.